views/pages/camps/index_user.jade

```diff
@@ -17,12 +17,12 @@ block content
                                     label=t('camps:new.name_en')
                                     input.hidden(id='join_camp_request_join_user_id', type='hidden', value='#{user.attributes.user_id}')
                                     select(type="text", id='join_camp_request_camp_id', class="form-control", name="camp_name_en", autofocus="true", required)
-                                        option(ng-repeat="camp in camps", value="{{camp.id}}") {{camp.camp_name_en}}
+                                        option(ng-repeat="camp in camps | filter:{__prototype: '!art_installation'}", value="{{camp.id}}") {{camp.camp_name_en}}
```

Comment:
Better to be explicit on the kinds of values you expect here, and not implicit.
Don't forget there may be other types  as well beside those two

Reply:
thought you asked to only remove art_installation camps from this dropdown.

If it's more complex, we should consider a different approach.

Reply:
@Aroundafur - Try searching for an option of passing an array of values to the filter param and pass the correct values (you can store it in a var in angularJS side)